packages/docusaurus-theme-classic/src/theme/NavbarItem/styles.module.css

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.dropdownButton {
+  font-size: inherit;
+  border: none;
+  background: none;
+  color: var(--ifm-navbar-link-color);
+  font-weight: var(--ifm-font-weight-semibold);
+  cursor: pointer;
+  padding: 0;
+}
+
+.dropdownButton:hover {
+  color: var(--ifm-navbar-link-hover-color);
+}
+
+.dropdownButton::after {
+  display: inline-block;
+  border-color: currentColor transparent;
+  border-style: solid;
+  border-width: 0.4em 0.4em 0;
+  content: '';
+  margin-left: 0.3em;
+  position: relative;
+  top: 2px;
+  transform: translateY(-50%);
+}
```

Comment:
Because I used a native button, I needed to reset native styles. I guess everything from this file should be moved/refactored to this repo https://github.com/facebookincubator/infima 

I didn't reuse the `navbar__link` because they were some styles I didn't want, for example:

https://github.com/facebookincubator/infima/blob/79b4f86e4d161a8a61205ab0bc4676f97460f9ae/packages/core/styles/components/navbar.pcss#L135